### PR TITLE
[Polish] Polish mvn wrapper.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Polish mvn wrapper.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
Just trigger ci.

**Review by wolfboys — COMMENTED** [review]
hi GOODBOY008:
thanks for your contribution, I see the maven-wrapper.jar file has been deleted, is that so? Please confirm.

**Comment by GOODBOY008** [comment]
> hi GOODBOY008: thanks for your contribution, I see the maven-wrapper.jar file has been deleted, is that so? Please confirm.

Yes , I confirm. Script `mvnw`  will download `maven-wrapper.jar` when `maven-wrapper.jar` doesn't exsit.

**Comment by wolfboys** [comment]
> > hi GOODBOY008: thanks for your contribution, I see the maven-wrapper.jar file has been deleted, is that so? Please confirm.
> 
> Yes , I confirm. Script `mvnw` will download `maven-wrapper.jar` when `maven-wrapper.jar` doesn't exsit.

In fact, this file is less than 1M, I can not think of a reason to delete, re-download will have to face the risk of download failure. I don't recommend delete file.

**Comment by GOODBOY008** [comment]
> wrapper

Ok, I will add `maven-wrapper.jar` with version `3.1.1` , which was `3.1.0` before.

**Comment by wolfboys** [comment]
hi GOODBOY008:
Can you add the details of this pr?

**Comment by GOODBOY008** [comment]
Currently,Maven wrapper's scrips and jar is out date and `MavenWrapperDownloader.java` should be deleted.

File: mvnw [hunk 1]
```diff
@@ -212,40 +186,67 @@ else
...
-        curl -o "$wrapperJarPath" "$jarUrl"
+        [ $? -eq 0 ] || rm -f "$wrapperJarPath"
     else
         if [ "$MVNW_VERBOSE" = true ]; then
           echo "Falling back to using Java to download"
         fi
-        javaClass="$BASE_DIR/.mvn/wrapper/MavenWrapperDownloader.java"
-        if [ -e "$javaClass" ]; then
-            if [ ! -e "$BASE_DIR/.mvn/wrapper/MavenWrapperDownloader.class" ]; then
+        javaSource="$BASE_DIR/.mvn/wrapper/MavenWrapperDownloader.java"
```
Comment (wolfboys) [inline mvnw:234]: MavenWrapperDownloader.java is deleted, need to declare this variable?

Comment (GOODBOY008) [reply mvnw:234]: The scripts is genarated by command `mvn wrapper:wrapper -Dmaven=3.8.4` and consider multi conditions to cover different scenes.

Comment (GOODBOY008) [reply mvnw:234]: @wolfboys Pls cc.

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**